packages/schematics/angular/application/index.ts

```diff
@@ -179,6 +179,34 @@ function addAppToWorkspaceFile(options: ApplicationOptions, appDir: string): Rul
   }
 
   const sourceRoot = join(normalize(projectRoot), 'src');
+  let budgets = [];
+  if (options.strict) {
+    budgets = [
+      {
+        type: 'initial',
+        maximumWarning: '400kb',
```

Comment:
Let's set this to 500kb in v10 just so that we are not too Draconian too quickly. We'll need to do a bit more testing before we know how low can we go given the state of the library ecosystem.

E.g. angular.io is at ~460kb today and that's a highly optimized app which is not the norm today.

In v11 and beyond we can drop the budget to 400kb or even lower once we have more data/experience of what that looks like in the real world.

Reply:
Agreed and updated.